Ginger/Ginger/UserControlsLib/UCTreeView/UCTreeView.xaml.cs

```diff
@@ -536,7 +536,7 @@ public void FilterItemsByText(ItemCollection itemCollection, string txt, Cancell
                 // Need to expand to get all lazy loading
                 tvi.IsExpanded = true;
 
-                ITreeViewItem ITVI = (ITreeViewItem)tvi.Tag;
+                //ITreeViewItem ITVI = (ITreeViewItem)tvi.Tag;
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6400213)